src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-multilingual.sch

```diff
@@ -106,7 +106,7 @@
       <sch:let name="government-names" value="document(concat('file:///', replace(concat($thesaurusDir, '/external/thesauri/theme/GC_Org_Names.rdf'), '\\', '/')))"/>
 
       <sch:let name="organisationName" value="gco:CharacterString" />
-      <sch:let name="isGovernmentOfCanada" value="starts-with(lower-case($organisationName), 'government of canada;') or starts-with(lower-case($organisationName), 'gouvernement du canada;')" />
+      <sch:let name="isGovernmentOfCanada" value="starts-with(lower-case(normalize-space(tokenize($organisationName, ';')[1])), 'government of canada') or starts-with(lower-case(normalize-space(tokenize($organisationName, ';')[1])), 'gouvernement du canada')" />
```

Comment:
This should be the value from the settings `schema/iso19139.ca.HNAP/DefaultMainOrganizationName_fr` nad `schema/iso19139.ca.HNAP/DefaultMainOrganizationName_en`, rather than hard coded to "government of canada".

Reply:
@jodygarnett 
I don't disagree with this request however it is not part of this PR.  

I'm also not sure how to import these variables from layout/custom-fields.xsl  or from the settings as /root/gui/settings/schema/iso19139.ca.HNAP/DefaultMainOrganizationName_en

I suggest that you create a new issue for this request.